dev-db/mysql/mysql-5.0.83.recipe

```diff
@@ -1,11 +1,18 @@
-DESCRIPTION="SQL database server."
+SUMMARY="SQL Database server"
+DESCRIPTION="A very fast and reliable SQL database server"
 HOMEPAGE="https://www.mysql.com"
 SOURCE_URI="http://mirror.provenscaling.com/mysql/community/source/5.0/mysql-5.0.83.tar.gz"
 CHECKSUM_MD5="051392064a1e32cca5c23a593908b10e"
 REVISION="1"
 STATUS_HAIKU="untested"
 DEPEND="sys-libs/readline >= 5.2"
 
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+        mysql$secondaryArchSuffix = $portVersion"
```

Comment:
This recipe is still using the old format has the so maybe we should remove it from this PR and create a separate PR for it.
(The `STATUS_HAIKU` variable comes from the old format.)

BTW, that's a very good idea to have one commit per port. We'll be able to merge this PR with the "Rebase and merge" method.